Files/UserControls/NavigationToolbar.xaml

```diff
@@ -463,7 +463,7 @@
             <Grid
                 x:Name="ClickablePath"
                 Grid.Column="1"
-                MinHeight="32"
+                MinHeight="33"
```

Comment:
Nit, should the height of the path box be divisible by 4?

Reply:
Normally yes, but this appears to be an exception. I believe it's because of the border but it may be something else.